### DISCUSSION TITLE
App Bar / Nav Bar

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#123] [2 - Shaping]
The app/Nav Bar at the bottom of the app has two versions: one for facilitator-level accounts in a team and one for member-level accounts.

The facilitator-level account should have the following items (probably displayed by icon only due to space constraints). I am listing the items in the order they should appear from left to right:
Items in **bold** are only shown for facilitator-level accounts

- Activities Page (as described [here](https://github.com/GECO-Project-App/facilitation-app/discussions/122))
- **Exercise Catalog**
  - See all available exercises (each exercise card has a "Start" and "Add to Agenda" button)
  - "Create Agenda" / "Schedule Meeting" button
- Team Page
  - Baseball Card Collection (a grid of team members that links to a given team member's profile and contains their tutorial to me, personality, and skills assessment)
  **- Team management (invite, remove, and change roles for team members)**
  - Profile page (edit profile picture, display name, description, etc.)
- Settings Page
  - Change app language
  - submit feedback
  - log out, change password, change email address, etc.

The member-level app bar should have these items:
- Activities Page (as described [here](https://github.com/GECO-Project-App/facilitation-app/discussions/122))
- Team Page
  - Baseball Card Collection (a grid of team members that links to a given team member's profile and contains their tutorial to me, personality, and skills assessment)
  - Profile page (edit profile picture, display name, description, etc.)
- Settings Page
  - Change app language
  - submit feedback
  - log out, change password, change email address, etc.

**Comment by eliasfaltin** [comment]
@aripuguh let me know if this makes sense to you or not 🙏

**Reply by aripuguh** [reply]
Thank you so much for this @eliasfaltin ! Crystal clear ✨✨

**Comment by aripuguh** [comment]
@alice-ang @Cristian-Nass

**Comment by aripuguh** [comment]
App Bar is [ready](https://www.figma.com/design/jA7uB2CX76FYBxRzwmTDzA/Facilitation-App?node-id=479-980). @eliasfaltin 
<img width="167" alt="Screenshot 2024-10-21 at 5 23 33 PM copy" src="https://github.com/user-attachments/assets/336479bf-7bf3-48a3-8044-252c3265571f">
